static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -95,14 +103,16 @@ require.def(
                     this._registerEventHandlers();
```

Comment:
Can't move this.logger to init - results in TypeError because this.getCurrentApplication().getDevice is undefined.